documentation/asciidoc/computers/os/graphics-utilities.adoc

```diff
@@ -141,6 +141,15 @@ Returns the throttled state of the system. This is a bit-pattern - a bit being s
 Returns the temperature of the SoC as measured by its internal temperature sensor;
 on Raspberry Pi 4, `measure_temp pmic` returns the temperature of the PMIC.
 
+[.whitepaper, title="Extra PMIC features on Raspberry Pi 4 and Compute Module 4", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-004340-WP/Extra-PMIC-features-on-Raspberry-Pi-4-and-Compute-Module-4.pdf]
```

Comment:
This should probably go somewhere in the https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#power-supply section.

Reply:
Agreed.